var/spack/repos/builtin/packages/pism/package.py

```diff
@@ -106,7 +111,7 @@ def cmake_args(self):
             ('YES' if '+python' in spec else 'NO'),
             '-DPism_BUILD_ICEBIN=%s' %
             ('YES' if '+icebin' in spec else 'NO'),
-            '-DPism_USE_PROJ=%s' %
+            '-DPism_USE_PROJ4=%s' %
```

Comment:
See the comment I left in the last PR about this line

Reply:
Closing the PR since https://github.com/spack/spack/pull/20284#discussion_r548942426 has not been fixed in a while. Feel free to reopen and ping me if you want to finish this PR.